src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/Route.php

```diff
@@ -58,4 +63,20 @@ public function setType($type)
     {
         $this->type = $type;
     }
+
+    /**
+     * @return int
+     */
+    public function getCacheTimeInSeconds()
```

Comment:
I would put this method name in the `RouteInterface`, same for the setter. You would force to implement that method when using models not only for PHPCR